ckanext/xloader/jobs.py

```diff
@@ -276,9 +280,11 @@ def _download_resource_data(resource, data, api_key, logger):
     data['datastore_contains_all_records_of_source_file'] = False
     which will be saved to the resource later on.
     '''
-    # check scheme
+ 
     url = resource.get('url')
-    url_parts = urlsplit(url)
+    url = modify_resource_url(url)
```

Comment:
I'm still not happy about this being applied to every resource download. How is this meant to handle cases where we _want_ to download from an external site?